Configs/.config/hypr/scripts/swwwallpaper.sh

```diff
@@ -1,5 +1,10 @@
 #!/usr/bin/env sh
 
+lockFile="/tmp/hyrpdots$(id -u)swwwallpaper.lock"
+ [ -e "$lockFile" ] &&  notify-send -a "Hyprdots" "An instance of the script is already running..." && exit  1 
```

Comment:
I think notification is not required here, as it just be spammed while changing wallpaper quickly.